hardware_interface/include/hardware_interface/hardware_info.hpp

```diff
@@ -52,7 +52,7 @@ struct HardwareInfo
    */
   std::vector<ComponentInfo> joints;
   /**
-   * \brief map of joints provided by the hardware where the key is the joint name.
+   * \brief map of sensors provided by the hardware where the key is the joint name.
```

Comment:
For the sensors one wants to use links sometimes, e.g., `robot_tcp` for an FTS
```suggestion
   * \brief map of sensors provided by the hardware where the key is the joint or link name.
```

Reply:
Good catch